apps/nova/src/pages/Settings.tsx

```diff
@@ -248,16 +248,24 @@ const Settings: React.FC = () => {
                     )}
                     checked={false}
                     disabled={false}
-                    onChange={() => alert("switch change")}
+                    onChange={() =>
+                      toast.success({
+                        title: intl("settings.notifications.comming-soon"),
+                      })
+                    }
```

Comment:
We can make the code more DRY by extracing this logic into a shared function and call it twice instead of writing it twice. We can call it `notifyComingSoon` (don't forget to wrap it with `useCallback`).